File: atcoder/go/abc229/a.go
package main

import (
	"fmt"
)

func main() {
	var s1, s2 string
	fmt.Scan(&s1)
	fmt.Scan(&s2)
	if (s1 == "#." && s2 == ".#") || (s1 == ".#" && s2 == "#.") {
		fmt.Println("No")
	} else {
		fmt.Println("Yes")
	}
}

File: atcoder/java/abc229/a.java
import java.util.*;

class Main {
    public static void main(String[] args) {
        try(Scanner sc = new Scanner(System.in)) {
            String s1 = sc.next();
            String s2 = sc.next();
            if((s1.equals(".#") && s2.equals("#.")) || (s1.equals("#.") && s2.equals(".#"))) {
                System.out.println("No");
            } else {
                System.out.println("Yes");
            }
        }
    }
}